dataCollection/sensors/TimeOfFlightSensor.java

```diff
@@ -257,7 +258,7 @@ public int getAddress(){
      */
     protected void update() {
         // wait for device to be ready for range measurement
-        // Logger.consoleLog("Indexer read: %s");
+        ConsoleLogger.consoleLog("Indexer read: %s");
```

Comment:
why do we want to revive this console logging?

Reply:
bump

Reply:
we don't